free-3.md

```diff
@@ -57,12 +67,15 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+I suspect, but am not sure, that the bulk of this project would be the language decisions. Giving users the flexibility to design their own games, but also making an engine that works for all of these specifications would be very tricky. The engine itself would be fairly simple, expectimax is pretty easy to write. 
 
```

Comment:
I agree that you'd have to spend a lot of time on language design.  Maybe I'm not properly understanding how your engine would work, but when I picture this DSL's inner workings it seems pretty complex to me.  Getting potentially unforseeable types of rules to work properly seems difficult.